src/main/java/org/quantumbadger/redreader/account/RedditAccountManager.java

```diff
@@ -206,17 +210,18 @@ private synchronized void reloadAccounts(final SQLiteDatabase db) {
 			while(cursor.moveToNext()) {
 
 				final String username = cursor.getString(0);
+				final String profileImage = cursor.getString(1);
 
 				final RedditOAuth.RefreshToken refreshToken;
-				if(cursor.isNull(1)) {
+				if(cursor.isNull(2)) {
```

Comment:
Would have been better to add this new field at the end of the query, to avoid updating all the other numbers?